doc/cephadm/index.rst

```diff
@@ -21,21 +21,13 @@ either via the Ceph command-line interface (CLI) or via the dashboard (GUI).
 versions of Ceph.
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 2
 
     compatibility
     install
     adoption
     host-management
-    mon
-    osd
-    rgw
-    mds
-    nfs
-    iscsi
-    custom-container
-    monitoring
-    service-management
+    service-management <services/index>
```

Comment:
When I checked read the docs for this the name for this section in the sidebar is now 'service-management' rather than "Service Management" like it had been previously

![image](https://user-images.githubusercontent.com/47704447/134337649-53d20856-96e7-4518-8213-7f31cc2c568a.png)

without these changes:

![image](https://user-images.githubusercontent.com/47704447/134337776-f5a61c61-9bab-42d5-8bb5-14efd41cca42.png)

otherwise, LGTM